src/uv_tcp_listen.c

```diff
@@ -275,51 +285,126 @@ static void uvTcpListenCb(struct uv_stream_s *stream, int status)
     assert(rv != 0);
 }
 
+/* Do bind/listen call on the tcp handle */
+static int UvTcpBindListen(struct uv_tcp_s *listener, struct sockaddr *addr)
```

Comment:
Nit: should be callsed `uvTcpBindListen`, capital case is only used for exported symbols, not for module-level static ones.

Reply:
If I remove the else block a lot of the integration unit tests (e.g. all tcp_connect, append, write, listen,...) will fail as the close_cb will not be invoked.

Reply:
Was not aware of this convention. Fixed it for all newly added helper function in uv_tcp_listen.c